### ISSUE TITLE
Columns block: Vertically align middle centres content in the editor

### ISSUE THREAD

**Issue by alaczek** [WordPress/gutenberg#20488]
**Describe the bug**
In the columns block, vertically centring content within a column, centres the content horizontally as well. This happens in the editor only, on the fronted the styling is applied correctly. Tested with TwentyTwenty theme, but was able to reproduce with other themes as well.

**To reproduce**
1. In the editor add columns block. Add content to a column.
2. Click on "Vertically Align Middle" option. Notice how the column content is both vertically and horizontally centred. 
3. View the post/page. Notice how on the frontend column content is only vertically centred.

**Expected behavior**
For the editor to only vertically centre content when using the "Vertically Align Middle" of the columns block. 

**Screenshots**
Video of the issue: https://cloudup.com/cJtnUnLCrvp

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.6
 - Browser: Chrome 80.0.3987.10
 - WordPRess 5.3.2. theme: TwentyTwenty

**Comment by jorgefilipecosta** [comment]
Hi @alaczek thank you for reporting this issue. Are you using "WordPress 5.3.2" with or without the Gutenberg plugin installed? If the Gutenberg plugin is installed, what version you used during your tests?
Thank you in advance for the additional information you may be able to provide.

**Comment by blt-sean** [comment]
Hi @jorgefilipecosta, I'm having a similar problem but I think it's down to an inconsistency with the way the data is output as a class name. On the back-end the text is "Vertically Align Middle", yet the value returned is 'center'. For themes using 'center' as horizontal and 'middle' as vertical positioning it can cause a few style issues - might this be causing a problem for @alaczek ?

Also, should I flag this as a separate ticket?

**Comment by alaczek** [comment]
@jorgefilipecosta I used WordPress without the Gutenberg plugin.

@blt-sean you might be right. When picking "Vertically Align Middle" option, the class applied to the column is "is-vertically-aligned-center". When I manually swapped it to "is-vertically-aligned-middle" in the dev console alignment worked as expected in the editor (but not on the front end). So seems like the class name applied is not accurate, and front end style for "is-vertically-aligned-middle" is missing.
